NARFO_BE/NARFO_BE/Models/Action.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NARFO_BE.Models
+{
+    public partial class Action
```

Comment:
please specify which type of models these are (request/response) if they are entity framework models then i guess this is fine